### PR TITLE
xfconf: added missing value types

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Checking a bit further into `xfconf-query` code, it's been found that a number of possible values were missing.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
xfconf

### CODE REVIEW

File: plugins/modules/system/xfconf.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
       For array mode, use a list of types.
```
Comment (felixfontein) [inline plugins/modules/system/xfconf.py:43]:
```suggestion
      For array mode, use a list of types.
    - Support for C(uchar), C(char), C(uint64), and C(int64) has been added in community.general 4.8.0.
```

**Comment by felixfontein** [comment]
@russoz you need to rebase this one.

**Comment by russoz** [comment]
Yup, and I just screwed up the rebase. Couple of minutes.

**Comment by russoz** [comment]
@felixfontein done

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**